user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+uid: Managing_cloud-connected_nodes
+---
+# Managing the nodes of a cloud-connected DMS
+
+As of DataMiner Cloud Pack version 2.5.0, you can manage the nodes of a cloud-connected DMS in the DCP Admin app. You can upgrade the installed DxM versions on the nodes and remove nodes that have become obsolete.
+
+> [!NOTE]
+> If nodes in a DMS cannot be detected in the Admin app, this is likely because they are using an older version of the DataMiner Cloud Pack or because the DataMiner Cloud Pack is not yet installed. Download the latest version from [DataMiner Dojo](https://community.dataminer.services/downloads/) and install it on every node in the DMS.
+
+## Upgrading nodes to the latest DxM versions
+
+To upgrade the DxMs used by the nodes in a cloud-connected DataMiner System:
+
+1. In the Admin app, check whether the correct organization is mentioned in the header bar.
+
+1. If a different organization should be selected, click the organization selector in the top-right corner and select the organization in the list.
+
+   ![Organization selector](~/user-guide/images/CloudAdmin_Selector.png)
+
+1. Click *DataMiner Systems* in the pane on the left, and then select your DataMiner System in the main pane.
+
+1. Click the *Manage nodes* button at the top of the pane on the left.
+
+1. To upgrade all installed DxMs to the latest version, click *Install latest available versions*. Alternatively, to upgrade specific DxMs only, click the *Upgrade* button for those DxMs.
```

Comment:
Maybe clarify that this is only for all the dxms on that specific node. Each node will have its own *install lastest* button.